gravitee-apim-console-webui/src/management/api/general-info/api-general-info.component.ts

```diff
@@ -99,6 +104,7 @@ export class ApiGeneralInfoComponent implements OnInit, OnDestroy {
     private readonly permissionService: GioPermissionService,
     private readonly snackBarService: SnackBarService,
     private readonly matDialog: MatDialog,
+    private integrationsService: IntegrationsService,
```

Comment:
What do you think of keeping the same convention as everywhere in the console code and keeping the services with `readonly` ?